packages/dd-trace/src/appsec/index.js

```diff
@@ -10,7 +10,7 @@ const Gateway = require('./gateway/engine')
 const addresses = require('./addresses')
 const Reporter = require('./reporter')
 const web = require('../plugins/util/web')
-const { extractIp } = require('./ip_extractor')
+const { extractIp } = require('./../plugins/util/ip_extractor')
```

Comment:
```suggestion
const { extractIp } = require('../plugins/util/ip_extractor')
```

Reply:
Can't believe I missed that :laughing: nice catch